packs/spell-effects/spell-effect-shields-of-the-spirit-security.json

```diff
@@ -0,0 +1,58 @@
+{
+    "_id": "5AIV3cCLUt442Mmp",
+    "img": "icons/magic/defensive/shield-barrier-blue.webp",
+    "name": "Spell Effect: Shields of the Spirit (Security)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Security]</p>\n<p>You grant an ally a +1 status bonus to AC.</p><hr /><p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Greater Security]</p>\n<p>If your shield is raised, the ally with the companion shield gets the same bonus to AC your shield grants.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core 2"
+        },
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "greater-security-bonus:{item|origin.system.attributes.shield.raised}"
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "ac",
+                "type": "status",
+                "value": 1
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "parent:origin:item:tag:greater-security",
+                    "greater-security-bonus:true"
```

Comment:
true?

Reply:
It's using the Roll Option rule element set up right above. The effect doesn't have the `origin:shield:raised` roll option, so this was a way to tell wether the origin has their shield raised or not.